config/application.rb

```diff
@@ -27,15 +27,15 @@ class Application < Rails::Application
 
     # Set the number of seconds the timeout warning should occur before
     # login session is timed out.
-    config.session_timeout_warning_seconds = 120
+    config.session_timeout_warning_seconds = 150
     # Set the number of seconds in which to delay the start of the
     # PeriodicalQuery() call. Make sure the sum of this value and
     # session_timeout_warning_seconds is a multiple of 60 seconds.
-    config.session_check_delay             = 60
+    config.session_check_delay             = 30
     # Set the frequency of the PeriodicalQuery() call in seconds.
     # Make sure the sum of this value and session_timeout_warning_seconds
     # is a multiple of 60 seconds.
-    config.session_check_frequency         = 60
+    config.session_check_frequency         = 30
 
```

Comment:
Should we move these to `application.yml` to keep all config in the same place? Just a thought for future PR.